README.md

```diff
@@ -19,7 +25,7 @@ The `#todo-list` element contains the main content of the application, is bound
 
 Another key Alkali concept used here is contextualized variables. All of the variables references from the view component are actually classes of variables, they are not direct references to instances. They can be thought of as "generalized" variables, and they will be contextualized as soon as they are used in instantiated components. While the class-based references make it very clear exactly what will be used, this deferred contextualization gives great flexibility in how the instances are created and related to each other. We are using a [`TodoList`](src/TodoList.js) as the data model, and alkali will automatically create a singleton instance for the view by default. However, we can easily specify a relationalship so that we could create multiple instances a the TodoView component that each have their own data model instance. Or we can even create a mock/test TodoList and inject into the TodoView for testing purposes.
 
-We actually use this typo of relationship for the `Editing` variable. In the TodoView, the `Editing` variable is defined (by extending `Variable`), and then associating it with each row in the view. This means there is a distinct instance of this variable created for each row that is generated. These distinct instances can then easily carry the individual view state of each row, as it may toggle between the view and editing state. Again, we simple reference `Editing` anywhere we have interest in this variable, indepedently of the scope or relationship we define for the variable. 
+We actually use this typo of relationship for the `Editing` variable. In the TodoView, the `Editing` variable is defined (by extending `Variable`), and then associating it with each row in the view. This means there is a distinct instance of this variable created for each row that is generated. These distinct instances can then easily carry the individual view state of each row, as it may toggle between the view and editing state. Again, we simple reference `Editing` anywhere we have interest in this variable, indepedently of the scope or relationship we define for the variable.
```

Comment:
looks like Atom [removed the last space](https://www.diffchecker.com/Z7lLVOEE) here.